js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
```

Comment:
I suggest you fix the issues in index.html line 34
tag closed improperly, spelling error, should be
not
since you have an incorrect extra letter (z) in line 34 and this is not semantic, please fix it so that its semantic.